packages/compass-saved-aggregations-queries/src/components/delete-item-modal.tsx

```diff
@@ -33,4 +36,19 @@ const DeleteItemModal: React.FunctionComponent<DeleteItemModalProps> = ({
   );
 };
 
-export default DeleteItemModal;
+export default connect<
```

Comment:
This aligns the implementation with [connect more components to read data](https://redux.js.org/style-guide/style-guide#connect-more-components-to-read-data-from-the-store) approach